typescript/ws-protocol-examples/package.json

```diff
@@ -67,7 +67,7 @@
     "@foxglove/schemas": "^1.0.0",
     "@foxglove/wasm-lz4": "^1.0.2",
     "@foxglove/wasm-zstd": "^1.0.1",
-    "@foxglove/ws-protocol": "^0.4.0",
+    "@foxglove/ws-protocol": "^0.5.0",
```

Comment:
Would we want to bump this to 0.5.1 in a separate PR? I suppose we will also want to bump the version of the examples package and publish that too.

Reply:
I can also do it in this PR if you don't mind?